examples/add_shapes.py

```diff
@@ -0,0 +1,53 @@
+"""
+Display one shapes layer ontop of one image layer using the add_shapes and
+add_image APIs. When the window is closed it will print the coordinates of
+your shapes.
+"""
+
+import numpy as np
+from skimage import data
+from skimage.color import rgb2gray
+from napari import ViewerApp
+from napari.util import app_context
+
+
+with app_context():
+    # create the viewer and window
+    viewer = ViewerApp()
+
+    # add the image
+    viewer.add_image(data.camera(), name='astronaut')
```

Comment:
I think the label should be updated to be "camera" or something more relevant.

Reply:
fixed